src/dotnet/APIView/APIViewWeb/Client/css/site.scss

```diff
@@ -538,4 +538,18 @@ mark {
 
 .apiview-logo {
     width: 30px;
+}
+
+.approval-value {
+    font-weight: bold;
+}
+
+.approval-approved {
+    font-weight: bold;
+    color: green;
+}
+
+.approval-pending {
```

Comment:
There are already a `font-weight-bolder `style and the `text-success`, `text-danger` styles.